src/offline/ledgerhash.ts

```diff
@@ -1,5 +1,5 @@
 import * as _ from 'lodash'
-import hashes = require('ripple-hashes')
+import {computeLedgerHash as clh, computeTransactionTreeHash, computeStateTreeHash} from '../common/hashes'
```

Comment:
I'd rather see us rename some functions than to alias a global import to an abbreviation of itself. 

Based on line 25, it looks like this function is actually `computeLedgerHeaderHash`, which I'd recommend renaming the exported function to.